modules/sb-install-tsb-operator.adoc

```diff
@@ -15,13 +15,29 @@ The following procedure installs the {tsb-name} Operator using the
 web console.
 
 . Create a namespace.
+
 .. Navigate in the web console to *Administration* -> *Namespaces* and click *Create Namespace*.
-.. Enter `openshift-template-service-broker` in the *Name* field and click *Create*.
+
+.. Enter the following:
++
+--
+* `openshift-template-service-broker` in the *Name* field
+* `openshift.io/cluster-monitoring=true` in the *Labels* field
+--
++
+Then click *Create*.
 +
-NOTE: The namespace must start with `openshift-`.
+[NOTE]
```

Comment:
Consider moving this up right below where you say to fill in the name field. Someone might have clicked create before they read this note.